.github/workflows/rust.yml

```diff
@@ -161,7 +162,7 @@ jobs:
 
       - name: Start local Sui network
         run: |
-          sui start --no-full-node &
+          cargo run --bin sui start --no-full-node &
```

Comment:
This is unrelated change, right?

Reply:
oops, not sure how this got in there. Removing